app/heimdall/components/MediaPlayer/MediaPlayer.tsx

```diff
@@ -17,6 +17,7 @@ export const MediaPlayer = () => {
           crossorigin
           playsinline
           poster="/__demo__/View_From_A_Blue_Moon_Trailer-HD.jpg"
+          style={{ width: "100%" }}
```

Comment:
Inline-styles aren't my first choice, but for the purposes of this demo, it will suffice.